plugins/coral-plugin-viewing-options/client/containers/ViewingOptions.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import {connect} from 'react-redux';
+import {bindActionCreators} from 'redux';
+import ViewingOptions from '../components/ViewingOptions';
+import {openViewingOptions, closeViewingOptions} from 'coral-embed-stream/src/actions/stream';
+
+const mapStateToProps = ({stream}) => ({open: stream.viewingOption.open});
```

Comment:
This would ensure future evolution of talk, were we want to have multiple embeds with connected store, and some store restructuring will be required.